scripts/gen-deps-list

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+const DepTree = require('deptree')
+const semver = require('semver')
+const { getPackages } = require('./utils')
+const [, , rootPackageName, incrementLevel] = process.argv
+
+if (!['major', 'minor', 'patch'].includes(incrementLevel)) {
+  console.error('Usage: gen-deps-list <package-name> <increment-level>')
```

Comment:
```suggestion
const [, command, rootPackageName, incrementLevel] = process.argv

if (!['major', 'minor', 'patch'].includes(incrementLevel)) {
  console.error('Usage: %s <package-name> <increment-level>', command)
```

Reply:
I feel you get this output because you are on macOS.

What I get on my side:
```
Usage: ./scripts/gen-deps-list.js <package-name> <increment-level>
```

I suggest we keep this behavior as it follows what is done by GNU utils (e.g. `ls`).